docs/ocis/deployment/ocis_wopi.md

```diff
@@ -26,6 +26,10 @@ There are three oCIS app driver containers that register Collabora and OnlyOffic
 
 The last four containers are the WOPI server, Collabora and OnlyOffice.
 
+## Overview Image
+
+{{< figure src="/ocis/deployment/ocis_and_wopi_drawio.svg" >}}
```

Comment:
Is this correct? Shouldn't it be 
```suggestion
{{< figure src="/docs/ocis/deployment/ocis_and_wopi_drawio.svg" >}}
```

Reply:
This is a very good question. I ran it locally with `make -C . docs-serve` and it worked - where the command was executed in the docs folder. Any hints and tips are welcomed

Reply:
@kobergj @micbar 
After a discussion with @micbar, I did a test in the root folder of ocis with and without using `/docs` in the path running `make -C . docs-serve`.

Only when NOT using `/docs`, the image gets rendered. This means, images are relative to the `/docs` path.

Reply:
Ok great :+1: